Industrial-IoT.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29201.188
+# Visual Studio Version 17
```

Comment:
are we safe changing VS version in the solution now?

Reply:
I restored the old version for safety